src/app/EventManagement.h

```diff
@@ -419,6 +423,16 @@ class EventManagement
      */
     CHIP_ERROR EnsureSpaceInCircularBuffer(size_t aRequiredSpace);
 
+    /**
+     * @brief Iterate the event elements inside event tlv and mark the fabric index as 0 if matching with obsolete fabric index
+     * converted from apContext
+     *
+     * @param[in] aReader  event tlv reader
+     * @param[in] apContext  the obsolete fabric index
+     *
+     */
+    static CHIP_ERROR RemoveInvalidFabricCB(const TLV::TLVReader & aReader, size_t aDepth, void * apContext);
```

Comment:
What is aDepth?

Reply:
*  @param[in]     aDepth       The current depth into the TLV data. 
 referenced from https://github.com/project-chip/connectedhomeip/blob/master/src/lib/core/CHIPTLVUtilities.cpp#L49
 updated, thanks